### PR TITLE
Add README

### LINKED ISSUE #18: Add README
README should include the usual information including how to install, remove, and build from source.

### PR DESCRIPTION
**Pull request by ahoneybun**
This PR handles this issue: https://github.com/pop-os/hidpi-daemon/issues/18

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,36 @@
+# HiDPI Daemon
+
+This program is for maanging HiDPI and LoDPI monitors on X. This program is installed by default in Pop!\_OS and Ubuntu (if installed by System76 and can be added with [this article](https://support.system76.com/articles/system76-software)).
```
Comment (leviport) [inline README.md:3]: s/maanging/managing

File: README.md [hunk 2]
```diff
@@ -0,0 +1,36 @@
+# HiDPI Daemon
+
+This program is for maanging HiDPI and LoDPI monitors on X. This program is installed by default in Pop!\_OS and Ubuntu (if installed by System76 and can be added with [this article](https://support.system76.com/articles/system76-software)). 
+
+## Building
+
+### Installing depends
```
Comment (leviport) [inline README.md:7]: s/depends/dependencies I think?

**Comment by leviport** [comment]
Two small typos, but this looks great!

**Comment by ahoneybun** [comment]
Should be good now!

**Review by leviport — APPROVED** [review]

**Review by 13r0ck — APPROVED** [review]
Thank the lord for squash and merge

@leviport please squash and merge this, don't just merge

**Comment by leviport** [comment]
I fixed the checks so Eoan and Bionic aren't blocking merges. Looks like those weren't updated in a while.

**Pull request merged**